### PR TITLE
Report when build in progress

### PR DESCRIPTION
**Pull request by malford**
https://app.asana.com/0/11331512503351/13274569624686

After merge add this line to jenkins shell command before `virtualenv -q $VENV` statement:

```
curl "http://localhost:3000/jenkins/pre_build?user=percolate&repo=hotlanta&sha=$GIT_COMMIT&status=pending&job=$JOB_NAME&build=$BUILD_NUMBER"
```

### CODE REVIEW

File: lib/github.coffee [hunk 1]
```diff
@@ -150,18 +156,12 @@ class PullRequestCommenter extends GithubCommunicator
...
-      @commentOnIssue pull.number, @errorComment()
-      cb null, pull
-
   updateComments: (cb) =>
     async.waterfall [
       @getPulls
       @findMatchingPull
       @removePreviousPullComments
       @updateCommitStatus
-      @makePullComment
```
Comment (jamesob) [inline lib/github.coffee:164]: no more comment on failure?

Comment (malford) [inline lib/github.coffee:164]: yeah @lra said it was cool to kill that

Comment (lra) [inline lib/github.coffee:164]: @jamesob we already have those:
screen shot 2014-07-03 at 2 41 09 pm
And @malford told me it was complex to keep

**Comment by jamesob** [comment]
dr @malford, I think a constructor may be out of date (`succeeded` -> `state`): https://github.com/percolate/jennifer/blob/report_build_progress/lib/github.coffee#L99

**Comment by jamesob** [comment]
also there's a bunch of cruft left around leaving comments, e.g. https://github.com/percolate/jennifer/blob/report_build_progress/lib/github.coffee#L151

**Comment by malford** [comment]
@jamesob I added back the pull comments as I was able to get it working.

**Comment by jamesob** [comment]
cool, lgtm
@ianwestcott any chance we can pull this branch onto our qa environment to test before merging this into the mainline?

**Comment by ianwestcott** [comment]
I believe that's what @malford was already doing for development, yeah?

**Comment by malford** [comment]
I was not on our qa environment. I was just testing against a test personal repo with a vagrant box clone of our Jenkins.
The personal repo I was using is https://github.com/malford/fuzzy-lana
You can see these PRs to see the results of this Jennifer branch:
https://github.com/malford/fuzzy-lana/pull/3
https://github.com/malford/fuzzy-lana/pull/1

If needed I can spin this Vagrant box up again for more testing.

**Comment by jamesob** [comment]
cool, merging.

**Pull request merged**